Makefile.common

```diff
@@ -82,7 +82,8 @@ PROMU_VERSION ?= 0.13.0
 PROMU_URL     := https://github.com/prometheus/promu/releases/download/v$(PROMU_VERSION)/promu-$(PROMU_VERSION).$(GO_BUILD_PLATFORM).tar.gz
 
 GOLANGCI_LINT :=
-GOLANGCI_LINT_OPTS ?=
+GOLANGCI_LINT_OPTS ?= --go=1.18
+
```

Comment:
Please revert these changes. Makefile.common is synced from prometheus/prometheus.

Reply:
Fixed